HLTrigger/Configuration/python/HLT_75e33_cff.py

```diff
@@ -0,0 +1,352 @@
+    fragment.L1T_PFPuppiHT450off,
+    fragment.L1T_PFPuppiMET220off,
+
+    fragment.HLT_AK4PFPuppiJet520,
+    fragment.HLT_PFPuppiHT1070,
+    fragment.HLT_PFPuppiMETTypeOne140_PFPuppiMHT140,
+
+    fragment.L1T_PFHT400PT30_QuadPFPuppiJet_70_55_40_40_2p4,
+    fragment.L1T_DoublePFPuppiJets112_2p4_DEta1p6,
+
+    fragment.HLT_PFHT330PT30_QuadPFPuppiJet_75_60_45_40_TriplePFPuppiBTagDeepCSV_2p4,
+    fragment.HLT_PFHT200PT30_QuadPFPuppiJet_70_40_30_30_TriplePFPuppiBTagDeepCSV_2p4,
+    fragment.HLT_DoublePFPuppiJets128_DoublePFPuppiBTagDeepCSV_2p4,
+    fragment.HLT_PFHT330PT30_QuadPFPuppiJet_75_60_45_40_TriplePFPuppiBTagDeepFlavour_2p4,
+    fragment.HLT_PFHT200PT30_QuadPFPuppiJet_70_40_30_30_TriplePFPuppiBTagDeepFlavour_2p4,
+    fragment.HLT_DoublePFPuppiJets128_DoublePFPuppiBTagDeepFlavour_2p4,
+
+    fragment.L1T_SingleTkMuon_22,
+    fragment.L1T_DoubleTkMuon_15_7,
+    fragment.L1T_TripleTkMuon_5_3_3,
+
+    fragment.HLT_Mu50_FromL1TkMuon,
+    fragment.HLT_IsoMu24_FromL1TkMuon,
+    fragment.HLT_Mu37_Mu27_FromL1TkMuon,
+    fragment.HLT_Mu17_TrkIsoVVL_Mu8_TrkIsoVVL_DZ_FromL1TkMuon,
+    fragment.HLT_TriMu_10_5_5_DZ_FromL1TkMuon,
+
+    fragment.L1T_TkEm51,
+    fragment.L1T_TkEle36,
+    fragment.L1T_TkIsoEm36,
+    fragment.L1T_TkIsoEle28,
+    fragment.L1T_TkEm37TkEm24,
+    fragment.L1T_TkEle25TkEle12,
+    fragment.L1T_TkIsoEm22TkIsoEm12,
+    fragment.L1T_TkIsoEle22TkEm12,
+
+    fragment.HLT_Ele32_WPTight_Unseeded,
+    fragment.HLT_Ele26_WP70_Unseeded,
+    fragment.HLT_Photon108EB_TightID_TightIso_Unseeded,
+    fragment.HLT_Photon187_Unseeded,
+    fragment.HLT_DoubleEle25_CaloIdL_PMS2_Unseeded,
+    fragment.HLT_Diphoton30_23_IsoCaloId_Unseeded,
+    fragment.HLT_Ele32_WPTight_L1Seeded,
+    fragment.HLT_Ele26_WP70_L1Seeded,
+    fragment.HLT_Photon108EB_TightID_TightIso_L1Seeded,
+    fragment.HLT_Photon187_L1Seeded,
+    fragment.HLT_DoubleEle25_CaloIdL_PMS2_L1Seeded,
+    fragment.HLT_DoubleEle23_12_Iso_L1Seeded,
+    fragment.HLT_Diphoton30_23_IsoCaloId_L1Seeded,
+
+    ### Removed temporarily until final decision on L1T tau Phase-2
+    #fragment.L1T_DoubleNNTau52,
+    #fragment.L1T_SingleNNTau150,
+
+    fragment.MC_JME,
+    fragment.MC_BTV,
+    fragment.MC_Ele5_Open_Unseeded,
+    fragment.MC_Ele5_WP70_Open_Unseeded,
+    fragment.MC_Ele5_Open_L1Seeded,
+    fragment.MC_Ele5_WP70_Open_L1Seeded,
+    fragment.MC_Photon100_Open_Unseeded,
+    fragment.MC_Photon100EB_TightID_TightIso_Open_Unseeded,
+    fragment.MC_Photon100_Open_L1Seeded,
+    fragment.MC_Photon100EB_TightID_TightIso_Open_L1Seeded,
+
+    fragment.HLTriggerFinalPath,
+    fragment.HLTAnalyzerEndpath,
+])
+
+
+
+del fragment.offlineBeamSpot
+
+del fragment.me0RecHits
+del fragment.me0Segments
+
+del fragment.L1EGammaClusterEmuProducer
+#del fragment.L1TkElectronsEllipticMatchCrystal
+#del fragment.L1TkElectronsEllipticMatchHGC
+#del fragment.L1TkMuons
+#del fragment.L1TkPhotonsCrystal
+#del fragment.L1TkPhotonsHGC
```

Comment:
Now that you renamed added the above as hlt* and used them in HLT Phase 2, you should keep the ```del``` of the old instances as they should now no longer run in HLT Phase2.

Reply:
Actually, those `del` were deleting the **new** instances -- it was just that we were **redoing** the final L1T candidates at some point during development because we wanted to go with latest and greatest from L1, and at some point we switched to use the ones from the previous L1T step.

I think at this point it is better to go back to redoing them, in view of the up-and-coming new L1 emulator. And if we are redoing them, we do it with our own `hlt` prefix, as discussed above.

So, I comment the `del` lines, because the modules they are deleting do not exist at this point in time with those labels. If I uncomment them, it will crash.

Reply:
OK!